### DISCUSSION TITLE
Extra perks from retirements

### DISCUSSION THREAD

**Discussion by keinkurt** [Lurkars/gloomhavensecretariat#392] [Q&A] [answered]
Is there a mechanism to calculate the extra perks from previous retirements at character creation?

**Comment by Lurkars** [comment, marked as answer]
Just scroll down on the character sheet and look for the `Retirements` counter. Increase to your wished number and you should gain the perks. If for any other reasons, you can also use the `Extra Perks` count (currently this is only used for JOTL, where you gain this as scenario reward).

![image](https://github.com/Lurkars/gloomhavensecretariat/assets/2760570/d21494b4-0847-474a-b74b-1090479ba3d1)
